src/snowflake/snowpark/_internal/analyzer/analyzer_utils.py

```diff
@@ -1338,10 +1340,21 @@ def single_quote(value: str) -> str:
         return SINGLE_QUOTE + value + SINGLE_QUOTE
 
 
+def validate_quoted_name(name: str) -> str:
```

Comment:
This is duplicated[ ](https://github.com/snowflakedb/snowpark-python/blob/da4be4a1a47e645fb220b5f869b1ba2a8ca1011c/src/snowflake/snowpark/_internal/utils.py#L812)

Reply:
removed